lib/index.ts

```diff
@@ -330,11 +360,12 @@ export const Float64_BE: IToken<number> = {
  */
 export const Float64_LE: IToken<number> = {
   len: 8,
-  get(b: Buffer, off: number): number {
-    return b.readDoubleLE(off);
+  get(array: Uint8Array, offset: number): number {
+    return new DataView(array.buffer, array.byteOffset).getFloat64(offset, true);
```

Comment:
Now you are constructing a new DataView a lot, should you not create a helper function for it instead that also takes the byteLength into consideration as well?

```js
function toDV(uint8) {
  return new DataView(uint8.buffer, uint8.byteOffset, uint8.byteLength)
}
```